bspump/http/web/server.py

```diff
@@ -470,7 +471,15 @@ def render_html_output(self, json_data):
     def format_json_to_html(self, json_data):
         fields_html = []
         for key, value in json_data.items():
-            if isinstance(value, dict):
+            if key == "button":
```

Comment:
This is going to "polute" the namespace for jsons that we return. This would mean that it would no longer be possible to simply return a json that happens to have a key named `button`. Instead what you can do is create new class that inherits from `dict`. Something like:

```
class RichJSONOutput(dict):
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)
        self.buttons = []

    def add_button(name, on_click, button_id):
        self.buttons.append(
            Button(name=name,on_click=on_click, button_id=button_id)
        )
```

This would be, potentially the type of json_data. Then you could do `if hasattr(jsondata, "buttons")` to find out if it has buttons and then add the buttons that way.